TactilityCore/Source/StringUtils.cpp

```diff
@@ -79,7 +79,7 @@ std::string join(const std::vector<std::string>& input, const std::string& delim
 }
 
 std::string removeFileExtension(const std::string& input) {
-    auto index = input.find('.');
+    auto index = input.rfind('.');
```

Comment:
This suggestion would break the intended purpose:

```
/**
 * @return the first part of a file name right up (and excluding) the first period character.
 */
std::string removeFileExtension(const std::string& input);
```

The intent was to get rid of the entire extension in the case of `HelloWorld.app.elf`, so that I would get `HelloWorld` from it.

I will consider renaming the function at a later point, to avoid confusion about what "extension" exactly means, but it's fine as it is right now.

Reply:
I understand now, I thought it was to remove the last extension.